nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/src/main/java/org/apache/nifi/event/transport/configuration/ShutdownTimeout.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.event.transport.configuration;
+
+import java.time.Duration;
+
+public enum ShutdownTimeout {
+    QUICK(Duration.ofMillis(5000)),
+    DEFAULT(Duration.ofMillis(15000));
+
+    private final Duration value;
+
+    ShutdownTimeout(final Duration value) {
+        this.value = value;
+    }
+
+    public Duration value() {
```

Comment:
Recommend renaming this to `getDuration()` along with the `value` property.

Reply:
renamed